satnogsclient/scheduler/tasks.py

```diff
@@ -19,15 +19,21 @@
 from satnogsclient.receiver import SignalReceiver
 from satnogsclient.observer.commsocket import Commsocket
 from satnogsclient.observer.udpsocket import Udpsocket
-from satnogsclient.upsat import upsat_status_settings as status
 from satnogsclient.upsat import serial_handler
 from satnogsclient.upsat.gnuradio_handler import write_to_gnuradio, read_from_gnuradio
 from time import sleep
 
-
 logger = logging.getLogger('satnogsclient')
 
 
+def signal_term_handler(signal, frame):
+    dict = frame.f_locals
```

Comment:
``dict`` is a built-in type in python. Here you are overriding the dictionary definition with something else.